2018-06-30 09:46:51.191 7732 DEBUG oslo_db.sqlalchemy.engines [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-30 09:46:54.723 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:54.726 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:54.750 7732 DEBUG oslo_db.sqlalchemy.engines [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-30 09:46:54.961 7732 DEBUG nova.context [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),eb4ddf00-3e7d-4b20-9636-b3ee7d08c752(default) load_cells /usr/lib/python2.7/site-packages/nova/context.py:508
2018-06-30 09:46:54.962 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:54.964 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:54.965 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:54.966 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:54.989 7732 DEBUG oslo_db.sqlalchemy.engines [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-30 09:46:55.021 7732 DEBUG nova.quota [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Getting quotas for project c1c2f6957c9b4d8c928b9fba21698139. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-30 09:46:55.030 7732 DEBUG nova.quota [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Getting quotas for user 0b85e892a488459b93b8264361354626 and project c1c2f6957c9b4d8c928b9fba21698139. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-30 09:46:55.051 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:55.052 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:55.053 7732 WARNING nova.image.glance [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-30 09:46:55.104 7732 DEBUG nova.notifications.objects.base [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-06-30 09:46:55.110 7732 DEBUG nova.objects.instance [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lazy-loading 'tags' on Instance uuid 36e33986-2fc5-4133-b95f-0751a5164210 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-30 09:46:55.164 7732 DEBUG nova.conductor.manager [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] [instance: 36e33986-2fc5-4133-b95f-0751a5164210] block_device_mapping [BlockDeviceMapping(attachment_id=<?>,boot_index=0,connection_info=None,created_at=<?>,delete_on_termination=True,deleted=<?>,deleted_at=<?>,destination_type='local',device_name=None,device_type='disk',disk_bus=None,guest_format=None,id=<?>,image_id='5454d2f2-9a2e-4fad-a3c4-a3c57d381a6f',instance=<?>,instance_uuid=<?>,no_device=False,snapshot_id=None,source_type='image',tag=None,updated_at=<?>,volume_id=None,volume_size=None)] _create_block_device_mapping /usr/lib/python2.7/site-packages/nova/conductor/manager.py:968
2018-06-30 09:46:55.167 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:55.168 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:55.268 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 09:46:55.272 7732 DEBUG oslo_concurrency.lockutils [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Lock "eb4ddf00-3e7d-4b20-9636-b3ee7d08c752" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 09:46:55.585 7744 WARNING nova.image.glance [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-30 09:46:55.662 7744 DEBUG nova.notifications.objects.base [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-06-30 09:46:56.137 7747 WARNING nova.image.glance [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-30 09:46:56.186 7747 DEBUG nova.notifications.objects.base [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-06-30 09:46:56.436 7732 WARNING nova.image.glance [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] No protocol specified in for api_server 'http://10.0.20.23:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-30 09:46:56.453 7732 DEBUG nova.notifications.objects.base [req-945cab52-9dc3-49ef-9d1f-740c7f2c8e8d 0b85e892a488459b93b8264361354626 c1c2f6957c9b4d8c928b9fba21698139 - d59e89c8b67142f7aa585fac76528bfa d59e89c8b67142f7aa585fac76528bfa] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
